ui/perfherder/compare/ComparePageTitle.jsx

```diff
@@ -30,11 +30,13 @@ export default class ComparePageTitle extends React.Component {
       pageTitle: title,
       newPageTitle: title,
     });
+    window.document.title = this.props.defaultPageTitle;
     this.changeQueryParam(newPageTitle);
   };
 
   editpageTitle = newPageTitle => {
     this.setState({ newPageTitle });
+    window.document.title = newPageTitle;
```

Comment:
This isn't the right place for updating the tab title.
If a user starts editing the 'Perfherder Compare Revisions' header, then presses `Escape`, the header remains unchanged, but the tab title has changed, when it shouldn't have.

We'd rather update the tab title after the user presses `Enter`.